### DISCUSSION TITLE
No alerts on the alerts interface

### DISCUSSION THREAD

**Discussion by abdullahi0000** [Security-Onion-Solutions/securityonion#13331] [2.4]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

6

### RAM

16GB

### Storage for /

76.5GB

### Storage for /nsm

147.8

### Network Traffic Collection

span

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

hello everyone
am doing a final year project and am using security onion together with kali linux and metasploitable in vmware workstation, i want the security onion to monitor the traffic between the other two machines and detect any intrusions .

i followed the official youtube installation guide and i cant see any alerts on the alerts interface , but am seeing some detections on the detections and hunt interface and dashboard. regarding storage i allocated 200GB based on the youtube installation guide

please help me my project submission deadline is approaching soon, any guidance would be greatly appreciated. thank you in advance

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
Please review the Troubleshooting Alerts section of the documentation:
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts

**Reply by Sarra2611** [reply]
I have the same problem ever since i've upgraded to the version 2.4.80, No alerts are being generated no matter what i do. Even using the testmynids doesn't work. I tried even creating simple rules but no alerts. I have downloaded the agent on my machine, i can see events in the dashboard and hunt interface but no alerts are being generated. I checked the troubleshooting alerts in the documentation and i tried fixing the issue to no avail. Can you please help?

**Reply by dougburks** [reply]
@Sarra2611 From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.

**Comment by abdullahi0000** [comment]
Hi thanks for the reply however I’ve tried all those troubleshooting
methods but I still can’t receive no alerts, on the home_net configuration
i included the network am watching traffic for, my network set up is inside
the vmware theres 3 vms security onion , kali linux and metasploitable, so
i wanted the sec onion to monitor the traffic , they are all on the same
subnet address 192.168.253.0/24 NAT

**Reply by dougburks** [reply]
What was the outcome of the first step of the Troubleshooting Alerts section?
> If you have metadata enabled, check to see if you have metadata for the connections. Depending on your configuration, this could be Suricata metadata or [Zeek](https://docs.securityonion.net/en/2.4/zeek.html#zeek) metadata.

**Comment by abdullahi0000** [comment]
I am using zeek for meta data generation

**Reply by dougburks** [reply]
The point of that first step and my point in asking about it here is to see if you're actually getting the metadata that you expect:
- If you ARE getting the metadata that you expect, then that narrows the problem down to Suricata. 
- If you ARE NOT getting the metadata that you expect, then the traffic may not even be reaching your Security Onion box.

So to be very clear: if you go to SOC Dashboards and select the Connections dashboard, do you see all of the connections that you expect to see?

**Reply by abdullahi0000** [reply]
here some screenshots of the dashboard

![Screenshot 2024-07-20 083944](https://github.com/user-attachments/assets/a9138f67-56bd-44ea-9da3-dbee385cab32)

![Screenshot 2024-07-20 083357](https://github.com/user-attachments/assets/4628fcd4-f8de-4855-8f7a-dc36bb81efb5)
![Screenshot 2024-07-21 001229](https://github.com/user-attachments/assets/ed0ca9f1-f316-40f4-9c00-71d4175cc322)

![Screenshot 2024-07-20 105113](https://github.com/user-attachments/assets/18b14c3a-6d99-4b96-a010-bbfa70029e43)


![Screenshot 2024-07-20 083445](https://github.com/user-attachments/assets/6644777f-77c4-4ab2-b5b6-501d9f7b7b0b)
![Screenshot 2024-07-20 083538](https://github.com/user-attachments/assets/be0fd8af-bb0a-4fb3-a6c4-e1e61b492f9f)

**Reply by dougburks** [reply]
Looking at your screenshots, there are very few Zeek logs and they seem to be for multicast traffic. This would indicate that the unicast traffic that you're looking for is not reaching your Security Onion box. You'll need to figure out how to get the traffic to Security Onion.
